service/discovery/discovery_test.go

```diff
@@ -337,7 +337,7 @@ func (mockIsCloudResource) GetCreationTime() *time.Time {
 //	var ready = make(chan bool)
 //
 //	assessmentServer := standalone.NewAssessmentServer().(*service_assessment.Service)
-//	assessmentServer.ResultHook = func(result *assessment.Result, err error) {
+//	assessmentServer.resultHook = func(result *assessment.Result, err error) {
```

Comment:
Much commented out code here. Maybe remove or stage it?

Reply:
I didn't comment it out, but I think we do not need it any more? @oxisto @lebogg

Reply:
Yeah we can remove it